### DISCUSSION TITLE
Cache Docker Images with Docker Compose

### DISCUSSION THREAD

**Discussion by cha0sg0d** [ScribeMD/docker-cache#784] [Show and tell]
First of all, this repo was EXACTLY what I needed. Thank you so much for building it.

I use a couple of public Docker images for my local development, and I simply wanted to cache them when pulling from the Docker Registry.

If anyone else has this simple use case, check out this gist that worked for me. It combines [Docker Cache](https://github.com/marketplace/actions/docker-cache) and [Docker Compose](https://github.com/marketplace/actions/docker-compose-action).

[docker-compose-cache-github-action.yml](https://gist.github.com/cha0sg0d/43e34a1f25c92a7fcc6aea572ddae4fb#file-docker-compose-cache-github-action-yml)

**Comment by Kurt-von-Laven** [comment]
Glad to hear you have been getting value from docker-cache, and thank you for sharing your script with the community! You inspired me to enable discussions on this repository.
